ocamldoc/odoc_info.mli

```diff
@@ -135,7 +135,7 @@ module Name :
 (** Representation and manipulation of method / function / class / module parameters.*)
 module Parameter :
   sig
-    (** {3 Types} *)
+    (** {2 Types} *)
```

Comment:
The translation was indeed automated, and you are right I missed the fact that these headers were submodule headers and should retstart at `1`. Fixed, thanks.